tests/search/nearest_neighbor/nearest_neighbor.rb

```diff
@@ -71,10 +83,12 @@ def feed_docs
     # Inserting one and one document ensures the same (and deterministic) order of the documents on the content node.
     # This means we can change "targetNumHits" and get deterministic behaviour.
     for i in 0...10 do
+      txt = [ 17, 17, 17, 17, i ]
+      txt[i % 4] = (i % 5)
       doc = Document.new("test", get_docid(i)).
         # TODO: Collapse back to 'pos' when we can choose which algorithm to run in the query.
         add_field("pos", { "values" => [i, X_1_POS] }).
-        add_field("text", "txt #{i} with #{i % 2} and #{i % 3} plus #{i+1000}").
+        add_field("text", txt.join(' ')).
```

Comment:
I had to draw out the docs generated on a piece of paper to understand the test queries above. Perhaps we should create a function that feeds a single document, and call this explicitly 10 times to make the test data more clear?

Reply:
i just made the texts a bit more explicit, hope that's ok